### PR TITLE
[EOC-70] Update look and fell in app

### PR DESCRIPTION
**Pull request by adamklepacz**
Log in screen:
zrzut ekranu 2019-03-05 o 08 27 25
Dashboard:
zrzut ekranu 2019-03-05 o 08 24 55
Dashboard with notification:
zrzut ekranu 2019-03-05 o 08 25 04
List view:
zrzut ekranu 2019-03-05 o 08 26 12
Dialog:
zrzut ekranu 2019-03-05 o 08 26 31
Modal:
zrzut ekranu 2019-03-06 o 07 40 52
Archivized list:
zrzut ekranu 2019-03-05 o 08 27 01
Creation forms:
zrzut ekranu 2019-03-06 o 07 40 44

Archived lists view:
zrzut ekranu 2019-03-05 o 08 25 24
Cohort with lists; 
zrzut ekranu 2019-03-05 o 08 25 44
Cohort with no lists: 
zrzut ekranu 2019-03-05 o 08 35 05

### CODE REVIEW

File: client/src/common/components/DialogBox.jsx [hunk 1]
```diff
@@ -7,13 +7,17 @@ const DialogBox = ({ message, onCancel, onConfirm }) => (
...
-      </button>
-      <button className="dialogbox__button" onClick={onConfirm} type="button">
-        Confirm
-      </button>
+      <header className="dialogbox__header">
+        <h1 className="dialogbox__heading">{message}</h1>
+      </header>
+      <footer className="dialogbox__footer">
+        <button className="dialogbox__button" onClick={onCancel} type="button">
+          Cancel
```
Comment (alemikolo) [inline client/src/common/components/DialogBox.jsx]: Sometimes we have cancel button on the right side of the form/dialog and sometimes on the left... Maybe we can unify this in this task

File: client/src/common/components/ModalForm.scss [hunk 2]
```diff
@@ -9,6 +9,7 @@
   z-index: 10;
   min-width: 320px;
   max-width: 600px;
+  border-radius: 3px;
```
Comment (alemikolo) [inline client/src/common/components/ModalForm.scss]: In our landing page there is no border-radius, so I wonder if should use it here, maybe like in buttons all corners should be right angles...

File: client/src/modules/cohort/Cohort.scss [hunk 3]
```diff
@@ -1,36 +1,46 @@
 .cohort {
-  margin: 36px 0 0;
+  margin: 36px 0 25px 0;
+  box-shadow: 0 2px 10px rgba($shadow-color, 0.1);
+  border: 0;
+  background-color: $bg-color;
+  border-radius: 3px;
```
Comment (alemikolo) [inline client/src/modules/cohort/Cohort.scss]: consider my proposal regarding corners

File: client/src/modules/dashboard/Dashboard.scss [hunk 4]
```diff
@@ -0,0 +1,8 @@
+.dashboard {
+  &__form {
```
Comment (alemikolo) [inline client/src/modules/dashboard/Dashboard.scss]: Why you add form styles to dashboard .scss file? They was removed to dropdownForm. Dashboard shouldn't have form styles.

File: client/src/common/components/ModalForm.scss (hunk 2 above)
Comment (adamklepacz) [reply client/src/common/components/ModalForm.scss]: I have tried with no radius - it looks ugly in my opinion. I'm looking forward to get some feedback from Lukasz

**Pull request closed without merging**